apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -461,7 +461,7 @@ sophia_identity(_Cfg) ->
     %% Remote calling the identity contract
     IdC   = ?call(create_contract, Acc1, identity, {}),
     RemC  = ?call(create_contract, Acc1, remote_call, {}, #{amount => 100}),
-    42    = ?call(call_contract,   Acc1, IdC, main, word, 42),
+    42    = ?call(call_contract,   Acc1, IdC, main, word, {42}),
```

Comment:
Change `call_contract` to take a list of arguments?

Reply:
It would require a bit more redesign.